docs/features/UNIFIED_CANVAS.md

```diff
@@ -0,0 +1,96 @@
+The Unified Canvas is a tool designed to facilitate composition, enabling artists to use all available generation modes in a single unified workflow. This documentation will help explain the basics of using the Unified Canvas, as well as some advanced tools available to power users of the canvas.
+
+# Basics
+The Unified Canvas consists of two layers: The **Base Layer** and the **Mask Layer**, which can be toggled using the (Q) hotkey.
+
+### Base Layer
+The **Base Layer** is the image content currently managed by the canvas, and can be exported at any time to the gallery by using the **Save to Gallery** option. When the Base Layer is selected, the Brush (B) and Eraser (E) tools will directly manipulate the base layer. Any images uploaded to the canvas (or sent to the unified canvas from the gallery) will reset the canvas, and add the image as your initial base layer image.
+
+### Staging Area
```

Comment:
I suggest adding a diagram or screenshot to show people where the Staging Area and controls are located.